tests/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ if( TARGET parsec-ptgpp )
 endif( TARGET parsec-ptgpp )
 add_subdirectory(dsl/dtd)
 add_subdirectory(apps)
+add_subdirectory(taskpool_wait)
```

Comment:
This add_subdirectory must be moved inside the `TARGET parsec-ptgpp` block above

Reply:
only because the single test in the directory uses both PTG andDTD taskpools.